R/ui-legacy.R

```diff
@@ -305,29 +303,3 @@ ui_inform <- function(...) {
   }
   invisible()
 }
-
-is_quiet <- function() {
-  isTRUE(getOption("usethis.quiet", default = FALSE))
-}
-
-# Sitrep helpers ---------------------------------------------------------------
-
-hd_line <- function(name) {
-  ui_inform(crayon::bold(name))
-}
-
-kv_line <- function(key, value, .envir = parent.frame()) {
-  value <- if (is.null(value)) ui_unset() else ui_value(value)
-  key <- glue(key, .envir = .envir)
-  ui_inform(glue("{cli::symbol$bullet} {key}: {value}"))
-}
-
-
-# cli wrappers ------------------------------------------------------------
-
-ui_cli_inform <- function(..., .envir = parent.frame()) {
-  if (!is_quiet()) {
-    cli::cli_inform(..., .envir = .envir)
-  }
-  invisible()
-}
```

Comment:
This is from pre-existing, small scale efforts to start using cli and is no longer used anywhere.